src/Files.App/Utils/Cloud/Detector/GoogleDriveCloudDetector.cs

```diff
@@ -164,6 +171,11 @@ private static async Task Inspect(SqliteConnection database, string sqlCommand,
 			return googleDriveRegValueJson;
 		}
 
+		/// <summary>
+        /// Get the base file system path for Google Drive from the Registry.
+        /// For advanced "Google Drive for desktop" settings reference, see:
+        /// https://support.google.com/a/answer/7644837
+		/// </summary>
```

Comment:
```suggestion
		/// <summary>
        /// Get the base file system path for Google Drive from the Registry.
        /// </summary>
        /// <remarks>
        /// For advanced "Google Drive for desktop" settings reference, see:
        /// https://support.google.com/a/answer/7644837
		/// </remarks>
```

Being a bit fussy here, but remarks should be used for this kind of thing.